probe/endpoint/conntrack_test.go

```diff
@@ -0,0 +1,138 @@
+package endpoint_test
+
+import (
+	"bufio"
+	"encoding/xml"
+	"io"
+	"testing"
+	"time"
+
+	. "github.com/weaveworks/scope/probe/endpoint"
```

Comment:
"The import . form can be useful in tests that, due to circular dependencies, cannot be made part of the package being tested:"

The go-way is to have tests part of the package; but as we put the tests in separate packages using the dot import still applied.